R/calcSDPIndustry.R

```diff
@@ -0,0 +1,44 @@
+addIndustry_UE_from_FE <- function(output) {
+
```

Comment:
you should check here that the variables exist in you output and if not, run the needed reporting-scripts